### ISSUE TITLE
Thumb not centered on track; top of thumb on bottom of track

### ISSUE THREAD

**Issue by pkassonss** [jeanregisser/react-native-slider#70]
The thumb works, but is not centered on track, as sliders are supposed to be.

**Comment by efournival** [comment]
Same issue with `react-native-slider` 0.10.0.
The workaround is to add `thumbStyle={{ top: 22 }}` to the slider props.

**Comment by pkassonss** [comment]
Nope, the thumb, for me, still appears as a circle, just below the track:

`      <Slider value={this.state.quantity} onValueChange={(value) => this.setState({quantity: value})}
         minimumValue={1}
         maximumValue={40}
         step={1}
         trackStyle={customStyles2.track}
         thumbStyle={{ top: 22 }}
         minimumTrackTintColor='#30a935'
       />`

**Comment by efournival** [comment]
Can you provide a screenshot of the problem?
Adjusting `thumbStyle` should be enough although I thought they fixed the bug in the last version.

**Comment by pkassonss** [comment]
![screen shot 2017-07-07 at 12 01 14 pm](https://user-images.githubusercontent.com/25250587/27966137-28418dba-630c-11e7-829a-c07a41b93797.png)

**Comment by efournival** [comment]
Well, I updated the plugin wrongly (I'm a noob with NPM) and I was still on an old version of the plugin.
Updating it fixed the issue and there is no need for the `thumbStyle` style hack.

**Comment by pkassonss** [comment]
The last commit I see is June 3rd, and I just pulled this in after that, I believe.  My version is:  ^0.10.0
